docs/Updating-OCaml-Version.md

```diff
@@ -2,69 +2,94 @@
 
 ## Current version
 
-The most recent version that we use is Ocaml 4.12.1. We have not updated past that for the following
+The most recent version that we use is Ocaml 4.13.1. We have not updated past that for the following
 reasons (which should be re-confirmed periodically):
 
-- The package [`rtop`](https://opam.ocaml.org/packages/rtop/) does not support 4.13 or higher version.
+- The package [`sexplib`](https://opam.ocaml.org/packages/sexplib/) as of v0.15.0 has a bug preventing
+  compilation of expressions of the form
+
+    `[@deriving sexp] type t = u and ...`
+
+  where `u` is an identifier.
+- The package [`rtop`](https://opam.ocaml.org/packages/rtop/) did not support 4.13 or higher version.
```

Comment:
is this inaccurate?

Reply:
About `rtop`, maybe I misunderstood the preceding instruction to re-check periodically. That was an old blocker that has been fixed. I changed "does not support" to "did not support". We could just remove it. If so, do we also want to remove the step to check `make repl`?

Reply:
Ah yeah, this list should only be issues blocking upgrade to the next version of OCaml, if any, not a list of past issues. 

No, we should still check `make repl` whenever we upgrade.

Reply:
What about the `sexplib`  item? We only get OCaml 4.13.1 if we downgrade `sexplib` from 0.15 to 0.14, but it also takes several lower-level packages (e.g., `core`) with it.  Since ocaml's latest version is 4.14, it might not matter.

Ignoring the `sexplib` issue, `incr_dom` is blocking OCaml 4.14.0, so I'll add an item for that.

Also, I see a lot of uncapitalized c's.

Reply:
incr_dom 0.15 supports ocaml 4.14 i think?

Reply:
Yes, but it and a couple other packages depend on `js_of_ocaml` < 4.0.0 which depends on `ocaml` < 4.14.

Reply:
Bonsai still depends on incr_dom, and I vaguely recall something on the Jane Street blog about it being created specifically as their own high-level API for incr_dom, so I doubt they'd kill it suddenly or without notice.

Naively "upgrading" to Bonsai is straight forward, but we wouldn't benefit from e.g. automatic incrementalization without a major redesign.